tests/system/action/topic/test_create.py

```diff
@@ -56,11 +56,40 @@ def test_create_multiple_requests(self) -> None:
                 },
             ],
         )
-        self.assert_status_code(response, 400)
-        self.assertIn(
-            "Datastore service sends HTTP 400. The following locks were broken: 'list_of_speakers/meeting_id', 'list_of_speakers/sequential_number', 'topic/meeting_id', 'topic/sequential_number'",
-            response.json["message"],
-        )
+        if response.status_code == 202:
```

Comment:
Mmh, this can potentially happen with every request, right? Can't you just set the `THREAD_WATCH_TIMEOUT` to something high enough (or even better: 0 to disable it, if it's not too much work) so we don't need this kind of special handling? (Or alternatively use the header mentioned above)

I really don't like that random tests could potentially fail now because of this timeout when this has nothing to do with what we're trying to test, also adding this special handling to every test is no solution either.

Reply:
This test covers both possible cases and I think this is a really good test as an example, how backend-actions have to be handled. This is implemented in the client with https://github.com/OpenSlides/openslides-client/pull/1423.

Reply:
Having a specific test (or multiple) to test the action worker behaviour is good and needed, but as I see it, currently every test might fail randomly if it takes longer than 1s since the backend then returns a 202 instead of the code expected by the test (200/400/...). This is a problem and should be fixed before we merge this. My proposal (since implementing some kind of off-switch for the whole action worker is as you say more work to implement): Set the `THREAD_WATCH_TIMEOUT` to `-1` automatically in every test (by mocking it or overwriting it, your choice). This tells `lock.acquire()` to wait indefinitely (https://docs.python.org/3/library/threading.html#threading.Lock.acquire), which would basically be the same as disabling the action worker if I understand it correctly.

Reply:
done